docker/generator/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM golang:1.14.3-alpine
```

Comment:
nit: We have a bunch of docker artifacts that aren't in this docker subfolder; could we put this `generator` folder under `build`?